pkgs/misc/cups/drivers/fflinuxprint/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, lib, fetchurl, dpkg, autoPatchelfHook, cups }:
+let
+  debPlatform =
+    if stdenv.hostPlatform.system == "x86_64-linux" then "amd64"
+    else if stdenv.hostPlatform.system == "i686-linux" then "i386"
+         else throw "Unsupported system: ${stdenv.hostPlatform.system}";
+in
+stdenv.mkDerivation rec {
+  pname = "fflinuxprint";
+  version = "1.1.3-4";
+
+  src = fetchurl {
+    url = "https://support-fb.fujifilm.com/driver_downloads/fflinuxprint_1.1.3-4_amd64.deb";
+    sha256 = "434a81e20bc459ad74286b7a4a7815aab6b115e3f1210eb69d3ac5678e2fcab5";
```

Comment:
This is optional since it could be automatized treewide at some point.

The idea is to avoid using `sha256` as attribute and use `hash` everywhere.

Since there could be different hashing algorithms in use, this make sense.

Reply:
Done.